assets/scripts/store/reducers/settings.js

```diff
@@ -14,7 +14,13 @@ const initialState = {
 
   saveAsImageTransparentSky: false,
   saveAsImageSegmentNamesAndWidths: false,
-  saveAsImageStreetName: false
+  saveAsImageStreetName: false,
+  markerLocation: null,
```

Comment:
Let's create a new reducer for map state. I'll create walkthrough for this in a separate comment.